tests/integration/test_auth.py

```diff
@@ -79,21 +79,20 @@ def test_auth_can_create_authenticated_requests_sessions():
     assertions.assertTrue("Bearer" in session.headers["Authorization"])
 
 
-def test_auth_can_fetch_s3_credentials():
+@pytest.mark.parametrize("daac", earthaccess.daac.DAACS)
+def test_auth_can_fetch_s3_credentials(daac):
     activate_environment()
     auth = earthaccess.login(strategy="environment")
     assertions.assertTrue(auth.authenticated)
```

Comment:
Let's also convert this assertion:

```suggestion
    assert auth.authenticated
```